src/integration/pages/articles/afrique/__snapshots__/canonical.test.js.snap

```diff
@@ -3,3 +3,7 @@
 exports[`Canonical Articles Gist list item should match text 1`] = `"Lorem ipsum dolor sit amet, consectetur adipiscing elit."`;
 
 exports[`Canonical Articles Gist title should match text 1`] = `"Points clés"`;
+
+exports[`Canonical Articles Image Copyright BBC should match snapshot 1`] = `null`;
```

Comment:
As above, it seems like there's a problem with the image copyright test, which is causing the null snapshot here